component/src/main/java/io/siddhi/extension/execution/string/FillTemplateFunctionExtension.java

```diff
@@ -34,82 +34,88 @@
 import io.siddhi.query.api.definition.Attribute;
 import io.siddhi.query.api.exception.SiddhiAppValidationException;
 
+import java.util.HashMap;
+import java.util.Map;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
+import static io.siddhi.query.api.definition.Attribute.Type.OBJECT;
 import static io.siddhi.query.api.definition.Attribute.Type.STRING;
 
 /**
- * replaceAll(string, regex, replacement)
+ * fillTemplate(string, replacement)
  * Replaces each substring of this string that matches the given expression with the given replacement.
- * Accept Type(s): (STRING,STRING,STRING)
+ * Accept Type(s): (STRING,STRING,STRING) or (STRING,OBJECT)
```

Comment:
The input can be (STRING,STRING/INT/LONG/FLOAT/DOUBLE/BOOL,STRING/INT/LONG/FLOAT/DOUBLE/BOOL) or (STRING,OBJECT)